x/auth/client/cli/encode.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/flags"
 )
 
-// GetEncodeCommand returns the encode command to take a JSONified transaction and turn it into
+// GetEncodeCommand returns the encode command to take a JSON field transaction and turn it into
```

Comment:
What does `a json field transaction` mean? I don't think `a JSONified transaction` contains a typo, it's correct here.

Reply:
Opps, really? JSONified? I must have misunderstood, I thought it was just a typo.